src/main/resources/application.yaml

```diff
@@ -166,6 +166,8 @@ security:
       - "uploadHomeOfficeAppealResponse"
       - "uploadAdditionalEvidenceHomeOffice"
       - "uploadAddendumEvidenceHomeOffice"
+    caseworker-ia-iacjudge:
+      - "endAppeal"
```

Comment:
Could you please add a functional test for notifications sent when endAppeal happens by a judge?